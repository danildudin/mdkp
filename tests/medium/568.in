29 6 0
337 17 325 470 314 215 232 555 967 227 698 942 140 715 198 34 536 519 903 577 745 270 749 371 249 887 924 949 593 
503 945 144 817 832 917 290 211 911 938 234 692 264 152 717 123 934 243 925 768 746 759 164 738 717 940 847 604 593 
910 258 981 150 530 354 71 291 489 867 980 2 308 5 80 35 530 861 913 392 605 66 111 814 417 407 283 922 787 
196 2 256 670 651 741 364 186 177 913 324 891 545 401 574 796 822 933 349 458 841 328 766 321 567 125 582 488 313 
938 906 961 53 503 706 728 269 735 322 746 28 748 389 189 96 388 174 971 208 944 222 671 930 813 760 208 956 978 
281 334 623 334 948 874 394 979 110 627 401 818 676 859 111 97 695 799 19 52 122 781 985 834 417 871 346 128 832 
456 228 223 546 312 508 502 677 639 981 390 202 199 229 71 121 337 763 169 513 316 850 800 786 890 811 785 753 912 
9532 9310 5475 9625 9679 6614 
