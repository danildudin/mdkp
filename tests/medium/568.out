9562
13
0 2 8 10 11 13 18 19 20 25 26 27 28 
